lepton/const.go

```diff
@@ -21,19 +21,22 @@ const manifest string = `(
 const Version = "0.2"
 const OpsReleaseUrl = "https://storage.googleapis.com/cli/%v/ops"
 
+// .ops directory
+const OpsDir string = ".ops"
```

Comment:
This should be use $HOME/.ops

Reply:
Extended a little use of constant OpsDir. It's needed to determine "base" directory in HOME of user. For example, path to `packages` directory is `path.Join(home, OpsDir, "packages")`